code/game/objects/items/stacks/medical_packs.dm

```diff
@@ -125,6 +125,7 @@
 	heal_brute = 10
 	stop_bleeding = 1800
 	dynamic_icon_state = TRUE
+	parent_stack = TRUE
```

Comment:
I think that `parent_stack = TRUE` being moved up to /obj/item/stack/medical/ would be good, this would also prevent splints and tribal splints/any other stuff added in the future from stacking

Reply:
Good call. might as well nip it in the bud. Added in next commit.